extensions/amp-video/1.0/video-wrapper.js

```diff
@@ -0,0 +1,209 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as Preact from '../../../src/preact';
```

Comment:
There's two `Preact` imports here. I'd like to be able to destructure this import like below, but this will fail to build without the star `*` import. I might be doing something wrong here.

Reply:
Is it preferable to duplicate the import, or to destructure separately as `const {...} = Preact;`?